app/views/pregnancies/_notes.html.erb

```diff
@@ -1,12 +1,14 @@
 <section id="notes" class="notes tab-pane">
-	<div class="margin-bottom col-md-12">
-		<h2>Notes</h2>
-    	<div class="info-form-left col-md-6">
-			<%= bootstrap_form_for [pregnancy, note] do |f| %>
-		    <%= f.text_area :full_text, size: '20x10', placeholder: 'Enter notes here', hide_label: true %>
-		    <%= f.submit class: "btn btn-primary"  %>
-		    <% end %>
-    	</div>
+	<div class="margin-bottom">
+		<div class="row">
+			<div class="info-form-left col-sm-6">
```

Comment:
Probably no reason to not have this as a `col-sm-12`, right? Let's change it to 12 for clarity's sake.